src/RollbarSourceMapPlugin.js

```diff
@@ -12,25 +12,29 @@ class RollbarSourceMapPlugin {
     version,
     publicPath,
     includeChunks = [],
-    silent = false
+    silent = false,
+    ignoreErrors = false
   }) {
     this.accessToken = accessToken;
     this.version = version;
     this.publicPath = publicPath;
     this.includeChunks = [].concat(includeChunks);
     this.silent = silent;
+    this.ignoreErrors = ignoreErrors;
   }
 
   afterEmit(compilation, cb) {
     const errors = validateOptions(this);
 
     if (errors) {
-      compilation.errors.push(...handleError(errors));
+      if (!this.ignoreErrors) {
```

Comment:
Here were checking for errors in the config options such as missing the rollbar tokens. We might not want to ignore this since it's more of a programmer error (vs. an operational error) that should fail the build.  Thoughts?